src/frontends/onnx/frontend/src/place.cpp

```diff
@@ -267,7 +267,7 @@ ov::frontend::Place::Ptr PlaceOp::get_output_port() const {
 
 ov::frontend::Place::Ptr PlaceOp::get_output_port(int output_port_index) const {
     check_if_valid();
-    if (output_port_index < m_editor->get_output_ports(m_node).size()) {
+    if (output_port_index < (int)m_editor->get_output_ports(m_node).size()) {
```

Comment:
Just for consistency with the rest of the codebase, please use `static_cast` here and below instead of C-style cast.